src/products/android-l10n/adding_projects.md

```diff
@@ -1,21 +1,27 @@
 # Adding a new project to android-l10n
 
-Follow the steps below to add new projects to [android-l10n](https://github.com/mozilla-l10n/android-l10n). You will have to coordinate with android-l10n developers and a technical Localization EPM throughout the process.
+Follow the steps below to add new projects to [android-l10n](https://github.com/mozilla-l10n/android-l10n). You will have to coordinate with android-l10n developers, release engineers, and a technical Localization EPM throughout the process.
 
 ## Setting up the project in source and l10n repositories
 
-Before anything happens, check that the new project is properly localizable by following the directions in [this section](https://github.com/mozilla-l10n/documentation/blob/main/src/tools/pontoon/adding_new_project.md#adding-a-new-project-on-pontoon). Once the new project exists in its corresponding source repo and is properly localizable, it can then be added to the `android-l10n` repository.
+Before anything happens, check that the new project is properly localizable by following the directions in [this section](https://github.com/mozilla-l10n/documentation/blob/main/src/tools/pontoon/adding_new_project.md#adding-a-new-project-on-pontoon). Once the new project exists in its corresponding source repo and is ready, it can then be added to the `android-l10n` repository.
 
-Android developers need to create a TOML file in their code repository and under its corresponding project folder (example for Focus [here](https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/l10n.toml)), as well as as the file containing the source strings in `res/values/strings.xml` (example with Focus [here](https://github.com/mozilla-mobile/firefox-android/blob/c718159480512abf370abcfd5aaf54b1d81074d3/focus-android/app/src/main/res/values/strings.xml)). The `firefox-android` repository contains all [current android projects](https://github.com/mozilla-mobile/firefox-android/tree/main).
+Android developers need to create a TOML file in their code directory and under its corresponding project folder (example for fenix [here](https://hg.mozilla.org/mozilla-central/file/tip/mobile/android/fenix/l10n.toml)), as well as the file containing the source strings in `res/values/strings.xml` (example with fenix [here](https://hg.mozilla.org/mozilla-central/file/tip/mobile/android/fenix/app/src/main/res/values/strings.xml)). The `mobile/android` directory in `mozilla-central` contains all [current Android projects](https://hg.mozilla.org/mozilla-central/file/tip/mobile/android).
```

Comment:
Another thing that I missed in review: using "here" for a link is not good practice, you can use the previous part of the sentence for the link (you tend to do that frequently).